cmd/gardener-admission-controller/main.go

```diff
@@ -15,20 +15,22 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/manager/signals"
 
 	"github.com/gardener/gardener/cmd/gardener-admission-controller/app"
 	"github.com/gardener/gardener/cmd/utils"
+	"github.com/gardener/gardener/pkg/logger"
 )
 
 func main() {
 	utils.DeduplicateWarnings()
+	logf.SetLogger(logger.MustNewZapLogger(logger.InfoLevel, logger.FormatJSON))
 
 	if err := app.NewCommand().ExecuteContext(signals.SetupSignalHandler()); err != nil {
-		fmt.Println(err)
+		logf.Log.Error(err, "Error starting app")
```

Comment:
`Error starting app` is neither helpful nor specific. Please call the applications by their name. Otherwise the additional message has no benefit at all.

Reply:
Good point, fixed in https://github.com/gardener/gardener/pull/9463/commits/8987f5b6c42c371b24d9c4c807257b70b8747954

Reply:
I think the current proposal aligns with structured logging in general, where we provide additional context in the key/value pairs, and for that reason I've added the app name in a separate `app` key/value pair.

This also aligns with the existing [Gardener Logging](https://gardener.cloud/docs/gardener/logging/#messages) documentation.

Let me know if you think additional changes are needed, thanks!